pkg/actions/nodegroup/create.go

```diff
@@ -293,6 +294,9 @@ func (m *Manager) checkARMSupport(ctl *eks.ClusterProvider, clientSet kubernetes
 }
 
 func loadVPCFromConfig(provider api.ClusterProvider, cfg *api.ClusterConfig) error {
+	if cfg.VPC == nil {
+		return errors.New("VPC configuration required for creating nodegroups on clusters not owned by eksctl.")
+	}
 	if cfg.VPC.Subnets == nil || cfg.VPC.SecurityGroup == "" || cfg.VPC.ID == "" {
```

Comment:
We can probably single-line this. I think it is _just_ on the side of not being so long it's gross
```suggestion
	if cfg.VPC == nil || cfg.VPC.Subnets == nil || cfg.VPC.SecurityGroup == "" || cfg.VPC.ID == "" {
```

Reply:
@Callisto13 I was thinking about the same but i did not want to touch the existing code and also return error i think vpc.subnets, vpc.id, vpc.securityGroup will give error because vpc details are empty when putting it in single-line.